Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zbus_polkit"
-version = "3.0.1"
+version = "4.0.0"
```

Comment:
btw, if it's doing to be a major version, might want to take the opportunity to make [`subject_kind` ](https://github.com/dbus2/zbus_polkit/blob/main/src/policykit1.rs#L159) an enum type (see the example at the every end of example code [here](https://docs.rs/zvariant/latest/zvariant/#example-code)). This doesn't need to be part of this PR though.

Reply:
I am not sure it's a good idea if it doesn't have an `Other(kind)` variant. The spec isn't really strict about allowed values:

https://www.freedesktop.org/software/polkit/docs/latest/eggdbus-interface-org.freedesktop.PolicyKit1.Authority.html#eggdbus-if-structures-org.freedesktop.PolicyKit1.Authority

Reply:
The following kinds of subjects are _known_